odoo/api.py

```diff
@@ -976,8 +991,8 @@ def get_todo(self):
         """ Return a pair ``(field, records)`` to recompute.
             The field is such that none of its dependencies must be recomputed.
         """
-        field = min(self.all.todo, key=self.registry.field_sequence)
-        return field, self.all.todo[field][0]
+        field = next(iter(self.all.todo))
```

Comment:
There was an optimization here, to avoid triggering the recomputation of already-recomputed fields.